tools/test/h5dump/h5dumpgentest.c

```diff
@@ -24,109 +24,125 @@
 #include "h5test.h"
 #include "h5tools.h"
 
-#define FILE1      "tgroup.h5"
-#define FILE2      "tdset.h5"
-#define FILE3      "tattr.h5"
-#define FILE4      "tslink.h5"
+#define TESTFILE_DIR "testfiles/"
```

Comment:
"testfiles/" should not be prepended to these test file names. There shouldn't be any assumptions about where the test file generator programs are being run from; a "testfiles" directory may not exist (nor should we try to create it from wherever the generator programs are being run from).